resources/eventing/charts/nats/templates/10-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.global.install.enabled }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nats-operator
+  # Change to the name of the namespace where to install NATS Operator.
+  # Alternatively, change to "nats-io" to perform a cluster-scoped deployment in supported versions.
+  namespace: {{ .Values.global.namespace }}
```

Comment:
Namespace does not have to be defined. Helm will do this automatically.

Reply:
removed